ivy_tests/test_ivy/test_frontends/test_torch/test_utilities.py

```diff
@@ -0,0 +1,73 @@
+# global
+# import ivy
+import numpy as np
+from hypothesis import given, strategies as st
+
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.torch as ivy_torch
+
+
+# noinspection DuplicatedCode
```

Comment:
You shouldn't include this tag. Duplicated code means the code exists elsewhere in the codebase and should be referenced.